### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MaitryG**

### CODE REVIEW

**Review by MunjNayak — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,24 +8,50 @@ Member details:
...
 | --- | --- | --- | --- |
 | Vaibhav Sharma | 40197697 | [vaibhav3003](https://github.com/vaibhav3003) | Scrum Master |
-| Kalinga Keshari Swain | 40226333 | | Project Manager |
-| Munj Bhavesh Nayak | 40195590 | | Backend Developer and Tester | 
+| Kalinga Keshari Swain | 40226333 | [Makermav](https://github.com/Markermav)| Project Manager |
+| Munj Bhavesh Nayak | 40195590 | [MunjNayak](https://github.com/MunjNayak) | Backend Developer and Tester | 
 | Maitry Gajiwala | 40234167 | [MaitryG](https://github.com/MaitryG) | Backend Developer and Tester |
-| Naoreen Chowdhury | 40217016 | | Frontend Developer | 
+| Naoreen Chowdhury | 40217016 | [naoreen](https://github.com/naoreen)| Frontend Developer | 
 | Khushali Parmar | 40224984 | | Frontend Developer |
```
Comment (Markermav) [inline README.md:15]: add khushali github username

File: README.md [hunk 2]
```diff
@@ -8,24 +8,50 @@ Member details:
...
 
 
-<h2>2. Core Features </h2>
-
-<h2>3. Project Management Philosophy</h2>
-
-1. **Project scope**
-<h2>4. User Roles</h2>
+## 2. Core Features
+**1. Tracking and Reminders for Job Applications**
```
Comment (Markermav) [inline README.md:24]: providing users access/login is one of the core feature; please add that

File: README.md [hunk 3]
```diff
@@ -8,24 +8,50 @@ Member details:
...
+- They can upload portfolio samples, certifications, and attach links to social media or personal websites.
+- Employers can create company profile showcasing company values, job postings, company achievements, posts regarding new product launches, etc.
+
+ **4. Networking and Collaboration**
+ - Students can connect with industry professionals, mentors, and like-minded individuals through networking features.
+ - Explore user directories, engage in chat/messaging, and participate in industry-specific forums/groups.
+ - Collaborate on projects, share documents, and receive valuable feedback through collaboration tools.
+   
+## 3. Project Approach
+(Write the technologies used) TBD
```
Comment (Markermav) [inline README.md:49]: Once Munj finds the necessary technology most feasible for us; please add it here!

File: README.md [hunk 4]
```diff
@@ -8,24 +8,50 @@ Member details:
...
+
+## 4. Project Management Philosophy
+**Project scope**
```
Comment (Markermav) [inline README.md:52]: Write the projects scope here!

File: README.md [hunk 5]
```diff
@@ -8,24 +8,50 @@ Member details:
...
+
+## 5. User Roles
```
Comment (Markermav) [inline README.md:54]: Remove this for the time being

File: README.md [hunk 6]
```diff
@@ -2,36 +2,25 @@
```
Comment (Markermav) [inline README.md:2]: Write the project title here.

**Review by Markermav — CHANGES_REQUESTED** [review]

**Review by naoreen — APPROVED** [review]

**Review by vaibhav3003 — APPROVED** [review]
Looks good, approving the request.

**Review by vaibhav3003 — APPROVED** [review]
Looks good, approving.

**Pull request merged**